build/buildall

```diff
@@ -65,15 +65,10 @@ function bloopInstall() {
       bloopTmpConfigDir="$bloopTmpDir/.bloop$bv"
       mkdir -p $bloopTmpConfigDir
       $MVN -B clean install \
-        ch.epfl.scala:bloop-maven-plugin:bloopInstall \
```

Comment:
Note for an orthogonal issue that we missed adding `--scala213` argument treatment in buildall's Bloop project generation, and it --scala213 is missing in the help message